docs/taskcluster-yml-v1.md

```diff
@@ -0,0 +1,161 @@
+---
+title: .taskcluster.yml version 1
+order: 20
+---
+
+Your main interface to Taskcluster-Github is via `.taskcluster.yml` in the root
+of your project. This is a YAML file that specifies the tasks to run and when.
+
+The format of the file is:
+
+```yaml
+version: 1
+policy:
+  pullRequests: ..
+tasks:
+  - (task definition)
+  - ...
+```
+
+The core of Taskcluster-Github's operation is this: when an event occurs on Github, such as a push or a pull request, it loads `.taskcluster.yml` from the commit specified in the event, renders it with JSON-e, and then calls `Queue.createTask` for each of the specified tasks.
+
+# Policies
+
+The `policy` property defines policies for what is allowed on the repository. Policies are always read from the default branch (generally `master`) of the repository. This prevents a malicious contributor from changing the policy applied to a pull request in the pull request itself.
+
+Policies are not rendered as a part of a task.
+
+##### Pull Requests
+
+Most projects prefer to run tasks for each pull request, so that the review process can take into account the task results. But if your project requires some secret data or uses some expensive service to test a pull request, then you probably do not want to run tasks for pull requests written by arbitrary contributors, but would still like to run tasks for PRs by project collaborators.
+
+The `pullRequests` policy controls this behavior:
+
+* `public` -- tasks are created for all pull requests.
+
+* `collaborators` (the default) -- tasks are created if the user who made the pull request is a collaborator on the repository.
+Github [defines
+  collaborators](https://developer.github.com/v3/repos/collaborators/#list-collaborators) as "outside collaborators, organization members with access through team memberships, organization members with access through default organization permissions, and organization owners."
+
+Example:
+```yaml
+policy:
+  pullRequests: public
+tasks: ...
+```
+
+# JSON-e Rendering
+
+The tasks defined in the YAML file are rendered using
```

Comment:
Let's be really precise here: The `tasks` property of `.taskcluster.yml` is rendered..".  Then users will know that they can say something like:

```yaml
tasks:
  $let
    x: 10
  in:
    ...
```

that is, `tasks` need not be a literal array, as long as it JSON-e renders to an array.